src/custom Hook/useFacilities.tsx

```diff
@@ -0,0 +1,28 @@
+import axios from "axios"
+import { useEffect, useState } from "react"
+import { toast } from "react-toastify"
+
+
+const useFacilities = () => {
+  const [facilities, setFacilities] = useState([{ "value": '', 'label': '' }])
+  let reqHeaders = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2NThhMTgyYjQ3ZWUyYjE0Zjk1NDY5OTAiLCJyb2xlIjoiYWRtaW4iLCJ2ZXJpZmllZCI6ZmFsc2UsImlhdCI6MTcwNDUzNjEyMCwiZXhwIjoxNzA1NzQ1NzIwfQ.p15lXfscJSFl8OJ4drIUj0vPPS3nO4L_U6iTbtwBdf8'
+  let Headers = { Authorization: reqHeaders }
+
+  {/*Get All Facilities */ }
+  const getAllFacilities = () => {
+    axios.get('http://154.41.228.234:3000/api/v0/admin/room-facilities', { headers: Headers }).then((response) => {
+      setFacilities(response?.data?.data?.facilities)
+      const newFacilities = response?.data?.data?.facilities
+      const facilities = newFacilities.map(({ _id: value, name: label }) => ({ value, label }))
```

Comment:
I think that you'll need to change this and return all data coming from the API, as we need all data in the facilities pagem but in adding room we need the label and value for each facility to be displayed in the Select component.
and after that you can make the mapping in the add-room page with the format you want.